package.json

```diff
@@ -10,7 +10,7 @@
     "clean": "rimraf dist",
     "lint": "eslint src/** test/**",
     "lint:fix": "npm run lint -- --fix",
-    "test": "mocha -R spec --compilers js:babel-core/register ./tests/setup.js ./test/**/*.spec.js",
+    "test": "mocha -R spec --compilers js:babel-core/register ./tests/setup.js ./tests/**/*.spec.js",
```

Comment:
Great catch! Can't believe this never got switched. I always use `test:cov`, so didn't notice.